### ISSUE TITLE
teaser=body deprecated?

### ISSUE THREAD

**Issue by eklem** [fergiemcdowall/norch-bootstrap#10]
After @mewwts last changes on `norch-bootstrap`, I think `teaser=body` doesn't have any useful application anymore. At least not out of the box.

**Comment by mewwts** [comment]
Oops, did I break something?

**Comment by eklem** [comment]
Not sure, but it's less understandable. Maybe only documentation. Now you auto-display all fields from template instead of defining which field to be teaser-text.

**Comment by eklem** [comment]
Think we need to understand a little better what/how to show stuff out of the box, and which tools to give the users when they develop their solutions further.

**Comment by mewwts** [comment]
Ah, I think @fergiemcdowall actually did that as it was weird imposing that documents needed to have certain fields in order to show.

**Comment by fergiemcdowall** [comment]
Yes @mewwts I have to take the blame for that change! @eklem - the teaser does still work though doesnt it..? Or is it broken?

**Comment by mewwts** [comment]
I thought this was a change for the better? :-)

**Comment by eklem** [comment]
I think it is for the better, but the `norch-bootstrap` adds the old `&teaser=body` to the link, and that is no use when it doesn't do anything. At least as I can see, since I have no key called body, and it would also not show up in the template?

**Comment by fergiemcdowall** [comment]
@eklem I agree that the current implementation is a bit clumsy, but Im not really sure how to improve it. How would you expect, teaser functionality to work in the default bootstrap?

**Comment by eklem** [comment]
We'll make one partial for "out of the box" and one to build upon? This I'll take a look at and come back with some more input.
